src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASKS = 101;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greetingMessage();
+        try {
+            readAndRespond();
+        } catch (DukeException e) {
+            System.out.println("--FATAL ERROR--");
+        }
+        exitMessage();
     }
+
+    private static void readAndRespond() throws DukeException {
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        Task[] listOfTasks = new Task[MAX_TASKS]; // 1-base indexing
+        while (!input.equals("bye")) {
+            String[] words = input.split(" ");
+            if (input.equals("list")) {
```

Comment:
Perhaps the conditionals in the `if` clauses could use a SLAP - https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard